pkg/kubectl/sorter.go

```diff
@@ -316,3 +317,76 @@ func (r *RuntimeSort) OriginalPosition(ix int) int {
 	}
 	return r.origPosition[ix]
 }
+
+type TableSorter struct {
+	field string
+	obj   *metav1beta1.Table
+}
+
+func (t *TableSorter) Len() int {
+	return len(t.obj.Rows)
+}
+
+func (t *TableSorter) Swap(i, j int) {
+	t.obj.Rows[i], t.obj.Rows[j] = t.obj.Rows[j], t.obj.Rows[i]
+}
+
+func (t *TableSorter) Less(i, j int) bool {
+	iObj := t.obj.Rows[i].Object.Object
+	jObj := t.obj.Rows[j].Object.Object
+
+	var iValues [][]reflect.Value
+	var jValues [][]reflect.Value
+	var err error
+
+	parser := jsonpath.New("sorting").AllowMissingKeys(true)
+	err = parser.Parse(t.field)
+	if err != nil {
+		panic(err)
```

Comment:
Maybe just `Fatalf`, `panic` when failing to match a field will be to scary for a user, don't you think?

Reply:
Will update, although originally I was mimicing the behaior of the runtimeSorter https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/sorter.go#L274